sys-fs/genfstab/genfstab-24.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Genfstab - generate output suitable for addition to an fstab file"
+HOMEPAGE="https://github.com/scardracs/genfstab https://man.archlinux.org/man/genfstab.8"
+SRC_URI="https://github.com/scardracs/${PN}/releases/download/${PV}/${P}.tar.gz"
```

Comment:
In proxy-maint we dislike this, 
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D_in_SRC_URI
https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0103

but since we're not dealing with this package I don't mind. Just a habit you should perhaps grow out of ;)

Reply:
Fixed